policy.md

```diff
@@ -154,27 +251,21 @@ For hierarchies used to issue S/MIME certificates:
 - 2.23.140.1.5.4.2
 - 2.23.140.1.5.4.3
 
-#### 5.1.1 ETSI ####
+#### 5.2.1 ETSI
 
 Audits conducted by an accredited Conformity Assessment Body (CAB) MUST have their Audit Attestation Letter (AAL) uploaded to the CAB’s website. CA Owners provide the URL to the AAL on the CAB’s website, and ALV will verify those URLs against a list of approved CAB websites. ETSI AALs MUST follow the latest version of the AAL template on the ACAB'c [website](https://www.acab-c.com/downloads/).
```

Comment:
Alternative language:  "An Audit Attestation Letter (AAL) issued by an accredited Conformity Assessment Body (CAB) MUST be downloadable from the CAB’s designated repository for such AALs, which must be on the CCADB's list of approved websites. CA Owners must populate the CCADB with the URL where such AAL can be downloaded over https."  Then we could delete "ALV will verify those URLs against a list of approved CAB websites." ?